packages/pwa-kit-build/src/configs/webpack/config.js

```diff
@@ -225,7 +227,14 @@ const client =
                 entry: {
                     main: './app/main'
                 },
-                plugins: [...config.plugins, new LoadablePlugin({writeToDisk: true})]
+                plugins: [
+                    ...config.plugins,
+                    new LoadablePlugin({writeToDisk: true}),
+                    analyzeBundle && getBundleAnalyzerPlugin({
+                        reportFilename: 'client-report.json',
+                        statsFilename: 'client-stats.json',
+                    })
```

Comment:
Our webpack config would export configurations for 5 bundles. But we used to generate output file with the same filename (each time replacing the file content). So I tweaked our config such that it will now generate 5 separate files.